openff/evaluator/properties/density.py

```diff
@@ -105,6 +109,76 @@ def default_equilibration_schema(
         calculation_schema.workflow_schema = schema
         return calculation_schema
 
+    @classmethod
+    def default_preequilibrated_simulation_schema(
```

Comment:
IIUC this or something similar should also be copied into `openff/evaluator/properties/dielectric.py`?

Reply:
Probably, but I haven't been computing and testing dielectrics (or any non-enthalpies of mixing + densities) so was thinking of adding this in a future version. Would it be better to add a method that raises a `NotImplementedError` here?